contributors/devel/e2e-node-tests.md

```diff
@@ -223,7 +223,7 @@ the bottom of the comments section.  To re-run just the node e2e tests from the
 `@k8s-bot node e2e test this issue: #<Flake-Issue-Number or IGNORE>` and **include a link to the test
 failure logs if caused by a flake.**
 
-The PR builder runs tests against the images listed in [jenkins-pull.properties](https://git.k8s.io/kubernetes/test/e2e_node/jenkins/jenkins-pull.properties)
+The PR builder runs tests against the images listed in 'jenkins-pull.properties'
```

Comment:
Instead of deleting invalid links, we should be correcting/fixing them.

In this case, I think the docs are in need of an overhaul. See https://github.com/kubernetes/community/issues/2759.

/cc @BenTheElder

Reply:
Oops, somehow github didn't show the comments above?!  :woman_facepalming: